.travis_Dockerfile_centos

```diff
@@ -0,0 +1,6 @@
+FROM centos
```

Comment:
Suggest to use tagged version with `centos:7`.

Reply:
All right. I will change it to `centos:7`.